parm/ccpp_v15.2_c96.nml.IN

```diff
@@ -0,0 +1,303 @@
+
+
+&gfdl_cloud_microphysics_nml
+  sedi_transport = .true.
+  do_sedi_heat = .false.
+  rad_snow = .true.
+  rad_graupel = .true.
+  rad_rain = .true.
+  const_vi = .F.
+  const_vs = .F.
+  const_vg = .F.
+  const_vr = .F.
+  vi_max = 1.
+  vs_max = 2.
+  vg_max = 12.
+  vr_max = 12.
+  qi_lim = 1.
+  prog_ccn = .false.
+  do_qa = .true.
+  fast_sat_adj = .true.
+  tau_l2v = 225.
+  tau_v2l = 150.
+  tau_g2v = 900.
+  rthresh = 10.e-6  ! This is a key parameter for cloud water
+  dw_land  = 0.16
+  dw_ocean = 0.10
+  ql_gen = 1.0e-3
+  ql_mlt = 1.0e-3
+  qi0_crt = 8.0E-5
+  qs0_crt = 1.0e-3
+  tau_i2s = 1000.
+  c_psaci = 0.05
+  c_pgacs = 0.01
+  rh_inc = 0.30
+  rh_inr = 0.30
+  rh_ins = 0.30
+  ccn_l = 300.
+  ccn_o = 100.
+  c_paut = 0.5
+  c_cracw = 0.8
+  use_ppm = .false.
+  use_ccn = .true.
+  mono_prof = .true.
+  z_slope_liq  = .true.
+  z_slope_ice  = .true.
+  de_ice = .false.
+  fix_negative = .true.
+  icloud_f = 1
+  mp_time = 150.
+/
+
+
+&interpolator_nml
+  interp_method = 'conserve_great_circle'
+/
+
+
+&namsfc
+       FNGLAC   = "global_glacier.2x2.grb",
+       FNMXIC   = "global_maxice.2x2.grb",
+       FNTSFC   = "RTGSST.1982.2012.monthly.clim.grb",
+       FNSNOC   = "global_snoclim.1.875.grb",
+       FNZORC   = "igbp",
+       FNALBC   = "global_snowfree_albedo.bosu.t126.384.190.rg.grb",
+       FNALBC2  = "global_albedo4.1x1.grb",
+       FNAISC   = "CFSR.SEAICE.1982.2012.monthly.clim.grb",
+       FNTG3C   = "global_tg3clim.2.6x1.5.grb",
+       FNVEGC   = "global_vegfrac.0.144.decpercent.grb",
+       FNVETC   = "global_vegtype.igbp.t126.384.190.rg.grb",
+       FNSOTC   = "global_soiltype.statsgo.t126.384.190.rg.grb",
+       FNSMCC   = "global_soilmgldas.t126.384.190.grb",
+       FNMSKH   = "seaice_newland.grb",
+       FNTSFA   = "",
+       FNACNA   = "",
+       FNSNOA   = "",
+       FNVMNC   = "global_shdmin.0.144x0.144.grb",
+       FNVMXC   = "global_shdmax.0.144x0.144.grb",
+       FNSLPC   = "global_slope.1x1.grb",
+       FNABSC   = "global_mxsnoalb.uariz.t126.384.190.rg.grb",
+       LDEBUG   =.false.,
+       FSMCL(2) = 99999
+       FSMCL(3) = 99999
+       FSMCL(4) = 99999
+       FTSFS    = 90
+       FSNOL    = 99999
+       FSICL    = 99999
+       FTSFL    = 99999,
+       FAISL    = 99999,
+       FVETL    = 99999,
+       FSOTL    = 99999,
+       FvmnL    = 99999,
+       FvmxL    = 99999,
+       FSLPL    = 99999,
+       FABSL    = 99999,
+/
+
+&fv_grid_nml
+  grid_file = 'INPUT/grid_spec.nc'
+/
+
+&nam_stochy
+  ntrunc = 766
+  lon_s = 1536
+  lat_s = 768
+  skeb = 0.3
+  iseed_skeb = 2019102712451
+  iseed_shum = 2019102712452
+  iseed_sppt = 2019102712453
+  skeb_tau = 21600.
+  skeb_lscale = 500000.
+  skebnorm = 0
+  skeb_npass = 30
+  skeb_vdof = 5
+  shum = 0.005
+  shum_tau = 21600.
+  shum_lscale = 500000.
+  sppt = 0.5
+  sppt_tau = 21600.
+  sppt_lscale = 500000.
+  sppt_logit = .true.
+  sppt_sfclimit = .true.
+  use_zmtnblck = .true.
+/
+
+&nam_sfcperts
+/
+
+&cires_ugwp_nml

```

Comment:
Should this namelist really be empty? When I try an empty cires_ugwp_nml with the SCM, I get a runtime error:

`At line 1575 of file /volumes/d1/grantf/code/gmtb-scm/ccpp/physics/physics/ugwp_driver_v0.F
Fortran runtime error: Index '1' of dimension 3 of array 'zpu' above upper bound of 0`

That is, it appears the default values for the cires_ugwp_nml lead to a runtime error.

Reply:
This should not be zero. Let me fix this in #11. @climbfuji